csharp/ql/test/resources/stubs/System.Web.cs

```diff
@@ -178,6 +178,37 @@ public class HttpRequest
         public string RawUrl { get; set; }
         public HttpCookieCollection Cookies => null;
         public bool IsAuthenticated { get; set; }
+        public NameValueCollection Form => null;
+        public NameValueCollection Headers => null;
+        public NameValueCollection Params => null;
+        public string UserAgent(string s) => null;
```

Comment:
Is `UserAgent` a property? https://learn.microsoft.com/en-us/dotnet/api/system.web.httprequest.useragent?view=netframework-4.8.1